app/views/streams/main_stream.html.haml

```diff
@@ -123,8 +123,7 @@
               = t("bookmarklet.heading")
           .content
             != t("bookmarklet.explanation", link: link_to(t("bookmarklet.post_something"), bookmarklet_code))
-
-        - if AppConfig.settings.paypal_donations.enable? || AppConfig.bitcoin_donation_address
+        - if AppConfig.settings.paypal_donations.enable? || AppConfig.settings.liberapay.enable? || AppConfig.bitcoin_donation_address
```

Comment:
We should create a method `donations_enabled?` in the `AppConfig` or `ApplicationHelper`.